src/dev/flang/be/c/C.java

```diff
@@ -605,6 +605,26 @@ private void createCode(CFile cf)
                                                  r.ret()))));
     var ordered = _types.inOrder();
 
+    // start of fileio related intrinsics
+    // fuzion.std.fileio.read
+    // currently logs console messages instead of proper returns in case of errors
+    cf.print
+      (CStmnt.lineComment("helper to read bytes from a file"));
+      cf.print
+        ("void "+CNames.FILEIO_READ._name+"(char* path, void* content, size_t size)\n"+
+          "{FILE *f = fopen(path, \"r\");\n"+
+          "if (f == NULL)\n"+
+          "{printf(\"error happened\");\n"+
+          "exit(0);}\n"+
+          "else {\n"+"size_t reading = fread(content, 1, size, f);\n"+
+          "if (reading == 0)\n"+
+          "printf(\"there is no more bytes to read\");\n"+
```

Comment:
please use { } to enclose code blocks after `if`, `else`, etc.

The `printf` here looks like early debug output. Please mark with a comment `// NYI: <reason>` and add a reason instead like `under development, need to remove this debug output` or similar.

Reply:
should the comment be in the generated c file or in C.java?